docs/source/ruletypes.rst

```diff
@@ -2588,7 +2588,7 @@ menu in your channel and configure an Incoming Webhook, then copy the resulting
 
 Optional:
 
-``ms_teams_alert_summary``: Summary should be configured according to `MS documentation <https://docs.microsoft.com/en-us/outlook/actionable-messages/card-reference>`_, although it seems not displayed by Teams currently, defaults to ``ElastAlert Message``.
+``ms_teams_alert_summary``: MS Teams use this value for notification title, defaults to ``Alert Subject``. You can set this value with arbitrary text if you want to display notification title other than default ``Alert Subject``.
```

Comment:
This should be revised to be more clear that it is going to use the `alert_subject` and optional `alert_subject_args`, if those values are provided, otherwise it will fallback to the rule name as the summary. I don't think it actually defaults to the string `Alert Subject`.

Reply:
Oh I was actually referring [https://elastalert2.readthedocs.io/en/latest/ruletypes.html#alert-subject](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#alert-subject)

Reply:
If you're referring to a section of the docs then replace the code formatting around "Alert Subject" with your hyperlink.

So instead of 
```
``Alert Subject``
```

you would use

```
`Alert Subject <https://elastalert2.readthedocs.io/en/latest/ruletypes.html#alert-subject>`_
```